Person.rb

```diff
@@ -0,0 +1,32 @@
+class Person
+  attr_reader :id, :name, :age
```

Comment:
- Optional
Kindly use the `attr_accessor` for both `name` and `age` instead of using custom setters. It would make your code clean and easy to read.